v2/googlecloud-to-googlecloud/pom.xml

```diff
@@ -35,6 +35,24 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.xolstice.maven.plugins</groupId>
+        <artifactId>protobuf-maven-plugin</artifactId>
+        <version>0.6.1</version>
```

Comment:
Nit any reason to duplicate this? It's already in v2/pom.xml.

If so, we should probably externalize to a variable in that pom and reuse.

Reply:
ok, doing that